database/sqlite3/sqlite3_test.go

```diff
@@ -181,3 +182,35 @@ func TestMigrateWithDirectoryNameContainsWhitespaces(t *testing.T) {
 	}
 	dt.Test(t, d, []byte("CREATE TABLE t (Qty int, Name string);"))
 }
+
+var pathTests = []struct {
```

Comment:
Move this inside `TestDbPathOutput` since it's not used by other tests.

Reply:
Done